MetaMask/metamask-mobile
# feat: add a footer to the in-app browser tab, containing information about the connected account and network

## **Description**

- Add a footer to the in-app browser tab, containing information about the connected account and network.

- The requirements [are in this issue](https://github.com/MetaMask/mobile-planning/issues/1634).

- The localizations originated from this PR https://github.com/MetaMask/metamask-mobile/pull/9474 I'm told another team picks it up and adds other languages [in this PR](https://github.com/MetaMask/metamask-mobile/pull/9428)


## **Related issues**

Fixes: It's a new feature not a fix

## **Manual testing steps**

1. Go to the browser tab
2. In the broswer's tab view, click on + to add a new tab
3. Again in the browser's tab view, click on the button on the right of the + sign to see the list of tabs
4. The tab should be displayed with the expected footer at the bottom like in the 'After' screenshot below.

## **Screenshots/Recordings**



| Before       | After        |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/MetaMask/metamask-mobile/assets/165834542/b35617ff-2afa-4dec-8a33-79984da04e6d"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/MetaMask/metamask-mobile/assets/165834542/10e13fa9-99cb-44af-b073-9d2da1fff617"> |


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
